### PR TITLE
Release prow script

### PR DESCRIPTION
**Pull request by TerryHowe**

### CODE REVIEW

File: release/prow.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+cat << EOF > awscliconfig
+[default]
+output=json
+region=${AWS_REGION:-${AWS_DEFAULT_REGION:-us-west-2}}
+role_arn=$AWS_ROLE_ARN
+web_identity_token_file=/var/run/secrets/eks.amazonaws.com/serviceaccount/token
+
+[profile release-account]
+role_arn=$ARTIFACT_DEPLOYMENT_ROLE_ARN
+region=${AWS_REGION:-${AWS_DEFAULT_REGION:-us-west-2}}
```
Comment (abhay-krishna) [inline release/prow.sh:38]: I'm thinking we should hardcode region to `us-east-1` here, since that is where the artifacts account is located.

Comment (TerryHowe) [reply release/prow.sh:38]: I'd be reluctant to change anything at this point. We normally run in us-west-2 and ECR public commands specify east if they need to already. S3 is region free.

Comment (TerryHowe) [reply release/prow.sh]: I made that us-east-1

**Review by abhay-krishna — APPROVED** [review]
/lgtm
/approve

File: release/prow.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+cat << EOF > awscliconfig
+[default]
+output=json
+region=${AWS_REGION:-${AWS_DEFAULT_REGION:-us-west-2}}
+role_arn=$AWS_ROLE_ARN
+web_identity_token_file=/var/run/secrets/eks.amazonaws.com/serviceaccount/token
+
+[profile release-account]
+role_arn=$ARTIFACT_DEPLOYMENT_ROLE_ARN
+region=${AWS_REGION:-${AWS_DEFAULT_REGION:-us-east-1}}
```
Comment (vivek-koppuru) [inline release/prow.sh:38]: Using AWS_REGION will be a problem here as the Pod already injects the region. Maybe we can add an `ARTIFACTS_AWS_REGION` env var to our prowjob and use that here.

**Review by vivek-koppuru — CHANGES_REQUESTED** [review]

**Pull request merged**